kustomize/README.md

```diff
@@ -0,0 +1,228 @@
+## INSTALLING TERRASCAN IN A KUBERNETES CLUSTER USING KUSTOMIZE
+
+This guide will help you install terrascan server inside your kubernetes cluster, as per your specific use case.
+We have covered four use cases in the sections below:
+
+  ##### Deploying Terrascan Server
+  Terrascan operating in http server mode.
+
+  #### Deploying Terrascan Server in TLS Mode
+  Terrascan operating in https server mode. This deployment is also a foundation for the terrascan webhook setup.
+
+  #### Deploying Terrascan Server for Remote Repository Scan
+  Terrascan in https server mode, with capability to scan remote repositories. This deployment can be used by an
+  argocd pre-sync hook that sends remote repository scan requests to the server.
+
+  #### Setting Up Terrascan Webhook
+  A Kubernetes Validating Webhook, that safeguards your cluster by denying the creation of kubernetes resources that
+  can cause potential security violations.
+
```

Comment:
Can we also add a cleanup section explaining how to delete k8s resources using kustomize?